ui/raidboss/data/06-ew/raid/p1s.txt

```diff
@@ -0,0 +1,106 @@
+57.7 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:65F[6789]:/
+
+# First tank mechanic
+72.1 "Pitiless Flail of Grace/Pitiless Flail of Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
+75.5 "True Flare/True Holy" sync / 1[56]:[^:]*:Erichthonios:661[01]:/
+
+# Second flail set
+91.1 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:65F[6789]:/
+105.3 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+
+# Tiles 1
+118.5 "Intemperance" sync / 1[56]:[^:]*:Erichthonios:661E:/
+122.6 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+134.2 "Intemperate Torment" sync / 1[56]:[^:]*:Erichthonios:661F:/
+135.3 "First Element" sync / 1[56]:[^:]*:Erichthonios:662[12]:/
+141.4 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+146.4 "Second element" sync / 1[56]:[^:]*:Erichthonios:662[123]:/
+151.6 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+157.3 "Third element" sync / 1[56]:[^:]*:Erichthonios:662[12]:/
+
+# Second tank mechanic
+167.9 "Pitiless Flail of Grace/Pitiless Flail of Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
+171.1 "True Flare/True Holy" sync / 1[56]:[^:]*:Erichthonios:661[01]:/
+
+# Cells 1
+188.2 "Shining Cells" sync / 1[56]:[^:]*:Erichthonios:6616:/
+194.5 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+207.8 "Aetherflail" sync / 1[56]:[^:]*:Erichthonios:6602:/
+208.6 "Powerful Fire/Light" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
+208.6 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:6DA6:/
+210.9 "Aetherflail" sync / 1[56]:[^:]*:Erichthonios:660D:/
+211.4 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:6DAD:/
+211.7 "Powerful Fire/Light" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
+224.2 "Pitiless Flail of Grace/Pitiless Flail of Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
+227.6 "True Flare/True Holy" sync / 1[56]:[^:]*:Erichthonios:661[01]:/
+229.7 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+242.9 "Aetherflail" sync / 1[56]:[^:]*:Erichthonios:6603:/
+243.7 "Powerful Fire/Light" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
+243.7 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:6DA7:/
+246.0 "Aetherflail" sync / 1[56]:[^:]*:Erichthonios:660A:/
+246.5 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:6DAC:/
+246.8 "Powerful Fire/Light" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
+255.1 "Shackles of Time" sync / 1[56]:[^:]*:Erichthonios:661C:/
+265.3 "Heavy Hand" sync / 1[56]:[^:]*:Erichthonios:6629:/
+270.0 "Inevitable Flame" sync / 1[56]:[^:]*:Erichthonios:6EC1:/
+270.0 "Out of Time" sync / 1[56]:[^:]*:Erichthonios:661D:/
+278.4 "Slam Shut" sync / 1[56]:[^:]*:Erichthonios:6617:/
+289.6 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+
+# Fourfold Shackles
+294.2 "Fourfold Shackles" sync / 1[56]:[^:]*:Erichthonios:6626:/
+303.2 "Chain Pain" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+308.2 "Chain Pain" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+313.2 "Chain Pain" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+318.2 "Chain Pain" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
```

Comment:
Here and elsewhere for similar repeated single abilities, to match other such repetitions in the repository:
```suggestion
303.2 "Chain Pain #1" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
308.2 "Chain Pain #2" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
313.2 "Chain Pain #3" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
318.2 "Chain Pain #4" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
```

Reply:
Didn't actually want to resolve this, thanks github.

I see this case, but is there another instance that's not `Gaoler's Flail` where this adjustment can be made? See https://github.com/quisquous/cactbot/pull/3969#issuecomment-1005320065 for why those aren't compacted/deduplicated/etc.

Reply:
Ah, right. After having seen it live myself, I get it better. I'll leave a comment directly on the Flail sequence